### DISCUSSION TITLE
1.18 Support

### DISCUSSION THREAD

**Discussion by Molotov6844** [jacobsjo/mc-datapack-map#9] [Q&A] [answered]
I am wondering if we can get 1.18.x support as this is the only tool that allows the usage of datapacks when seedfinding.

**Comment by jacobsjo** [comment, marked as answer]
I don't plan to properly support 1.18.x, sorry. 

However, if you just want biomes, you can still use the map with an 1.18.x datapack by doing the following:
- Disable the "Move location down to surface" option on the right
- Make sure the seed is set to the seed the datapack defines for the world (as that is determined by the datapack in 1.18.x)

**(I can't guarantee that this will give the correct result for every datapack)**

If the datapack has a version for 1.19 or 1.20 it might make sense to use that version in the map.

Properly supporting the terrain height and structures in 1.18 would require parsing old format noise settings, density functions, structure sets and structure features. While possible I think the effort is not worth it given that 1.18 is now 2 versions old.
